src/js/services/WebsiteService.ts

```diff
@@ -81,13 +89,15 @@ class WebsiteService {
         throw Error(`API version '${this.pluginOptions.api.version}' not recognized`)
       }
 
-      axios.defaults.headers['X-Origin'] = window.location.href;
       const result = await axios.post<Array<ITranslation>>(
         url,
         data,
         {
-          cancelToken: cancelToken
-        }
+          cancelToken: cancelToken,
+          headers: {
+            "X-Origin": window.location.href,
```

Comment:
Šis jau nebija iepriekš atrisināts?

Reply:
bija, te palika no otra aktīvā PR, ka netiek sūtīts kā default header, bet parasts, jo atšķirības nekādas, te tāpat tikai viens request nevis vairāki

Reply:
ok, aizmirsu par to PR